metron-docker/hbase/bin/init-hbase.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+export HBASE_CLASSPATH=`${HBASE_HOME}/bin/hbase classpath`
+./bin/hbase shell ./bin/init-commands.txt
+java -cp $HBASE_CLASSPATH:/usr/metron/$METRON_VERSION/lib/metron-data-management-$METRON_VERSION.jar org.apache.metron.dataloads.nonbulk.flatfile.SimpleEnrichmentFlatFileLoader -e /conf/enrichment-extractor.json -t enrichment -c cf -i /data/enrichments.csv
```

Comment:
I could not use the flatfile_loader.sh script because it depends on the hadoop cli.  This is not necessary to load data into HBase and isn't installed in the image.